mne/io/kit/tests/test_kit.py

```diff
@@ -60,7 +60,6 @@ def test_data():
                               stimthresh=1)
     assert_true('RawKIT' in repr(raw_py))
     assert_equal(raw_mrk.info['kit_system_id'], KIT.SYSTEM_NYU_2010)
-    assert_true(KIT_CONSTANTS[raw_mrk.info['kit_system_id']] is KIT_NY)
```

Comment:
You can still assert something about `info['kit_system_id']`, though, right?

Reply:
We are still asserting that the system-id is correct in the line above; I also added a check for the channel types (which are now read out form the file rather than hard-coded)

Reply:
Ahh you're right, missed that line